fcrepo-export/src/test/java/org/fcrepo/exporter/ExporterTest.java

```diff
@@ -146,6 +152,18 @@ public void testExportContainer() throws Exception {
         Assert.assertTrue(exporter.wroteFile(new File("target/rdf/rest/1.jsonld")));
     }
 
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
```

Comment:
Nope... this is apparently the way people use JUnit to test whether an exception has a given message.  It's completely non-intuitive to me.  Would it be acceptable to catch the exception in the test method and do the comparison there with assert statements?